flavors/standard-Exasol-8-cuda-ml/FLAVOR_DESCRIPTION.md

```diff
@@ -0,0 +1,20 @@
+# Details for template-Exasol-8-python-3.10-cuda-conda flavor
```

Comment:
Title doesn't fit the directory name and the directory name is still `standard...`

Reply:
Fixed